content/cookbook/tabs.md

```diff
@@ -0,0 +1,26 @@
++++
+title = "Tabs"
+date =  2020-07-20T12:41:40+01:00
+weight = 5
++++
+
+# How can I handle tabs in Sakuli?
+
+The simplest way to switch the tab is based on the position of the tab.
+{{<highlight javascript>}}
+let handles = await driver.getAllWindowHandles();
+await driver.switchTo().window(handles[1]);
+{{</highlight>}}
+
+To switch back to the first tab, we can then use:
+{{<highlight javascript>}}
+await driver.switchTo().window(handles[0]);
+{{</highlight>}}
+
+When you are finished with the tab, you can close it with `driver.close()`. Afterward, we need to change the focus back to one of the open tabs.
```

Comment:
We should be more precise which exact tab will be closed using `await driver.close();`
I assume, your given scenario is that you have two tabs open and switched to the second tab, did some stuff and now want to close it. If this is the case, it is not clearly transported to an inexperienced reader. The flow of the complete article is:
1. switch to a new tab
2. switch back to the first tab
3. close the current tab and then switch to the first tab again (?)

As the third scenario is not depending on the others, it would make sense to introduce the reader to the given scenario and explain precisely which effect the shown commands would have to that scenario.